pkgs/development/compilers/kcc/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, cmake, bison, flex, boost }:
+
+stdenv.mkDerivation rec {
+  pname = "kcc";
+
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    owner = "KnightOS";
+    repo = "kcc";
+    rev = version;
+    sha256 = "1cd226nqbxq32mppkljavq1kb74jqfqns9r7fskszr42hbygynk4";
+  };
+
+  nativeBuildInputs = [ cmake boost bison flex ];
+
+  meta = with stdenv.lib; {
+    homepage    = "https://knightos.org/";
+    description = "KnightOS C compiler";
+    license     = licenses.gpl2;
```

```suggestion
    license     = licenses.gpl2Plus;
```

`gpl2` deprecated, use `gpl2Only` or `gpl2Plus`. From a quick look at the source code, they use the 'or later wording', so this should be `gpl2Plus`.